src/master/develop/management.md

```diff
@@ -263,11 +484,19 @@ The snapshot creation does not necessarily take place upon each scan by prefix,
 After some time, several snapshots may be created (specified by flushAfter properties of the btree and the scan requests), the file backing the btree will hold several old snapshots. Thus the clean index process will dump to a different location only the latest snapshot but in this case also writing the unmodified nodes. Once that dump is done, the index folder is replaced by the new one.
 While the clean process is made, no data is indexed and there will be an extra disk space requirement due to the new dump. Once completed, a considerable disk space will be reduced by removing the previously indexed data (older snapshots).
 The btree and clean up process is something specific to indexing. And will not lock transaction processing as indexing is asynchronously generated.
+
 :::: tabs
 
 ::: tab Go
 ```go
-	client.CleanIndex(ctx, &emptypb.Empty{})
+	err = client.CompactIndex(ctx, &emptypb.Empty{})
+	// error handling
+
+	err = client.FlushIndex(ctx, &schema.FlushIndexRequest{
+		CleanupPercentage: 0.1,
+		Synced:            true,
```

Comment:
This is not documented - what's the difference between synced and not-synced flush?